apis/Google.Cloud.Storage.V1/Google.Cloud.Storage.V1/CopyObjectOptions.cs

```diff
@@ -128,6 +128,11 @@ public sealed class CopyObjectOptions
         /// </summary>
         public string UserProject { get; set; }
 
+        /// <summary>
+        /// Options to pass custom retry configuration for each API request.
```

Comment:
It's fine to use the term "custom" here and in other operation-specific options classes - but elsewhere it's just "the retry options" without differentiation between default or custom retry options.

(This comment is just intended to give some more background for other suggested changes.)